scripts/reborn_webui_v2_live_qa/run_live_qa.py

```
@@ -3154,6 +3154,22 @@ async def action(page: object) -> None:
+        # Extension-scoped OAuth deliberately rejects an absent installation.
+        # Exercise the same global install transition as the product UI before
+        # probing the OAuth start surface; do not manufacture per-user setup
+        # state inside the canary.
+        install_body = await _webui_json(
+            page,
+            "POST",
+            "/api/webchat/v2/extensions/install",
+            {"package_ref": {"kind": "extension", "id": "slack"}},
+        )
+        if install_body.get("success") is not True:
+            raise AssertionError(f"Slack install did not succeed: {install_body!r}")
```

Ruff (0.15.21)

[warning] Avoid specifying long messages outside the exception class

(TRY003)